source/blog/2018-09-28-the-ember-times-issue-66.md

```diff
@@ -35,7 +36,7 @@ Check it out on the [GitHub repo](https://github.com/mixonic/ember-cli-deprecati
 
 ---
 
-## [And Today's Special on The Router Menu: Query Params](https://github.com/emberjs/rfcs/pull/380)
+## [And Today's Special On The Router Menu: Query Params 🥡](https://github.com/emberjs/rfcs/pull/380)
```

Comment:
Hm I think Amy fixed that in a previous PR as the uppercase `On` didn't adhere to the titleCase format we were using usually: https://github.com/emberjs/website/blob/master/source/blog/CONTRIBUTING.md#writing-style-guide